x509-cert/Cargo.toml

```diff
@@ -15,7 +15,7 @@ edition = "2021"
 rust-version = "1.65"
 
 [dependencies]
-const-oid = { version = "0.9.2", features = ["db"] } # TODO: path = "../const-oid"
+const-oid = { version = "0.9.3", features = ["db"] }
```

Comment:
Won't compile until we publish a backport of #1129

Reply:
It's released